backend/src/apiserver/visualization/requirements.txt

```diff
@@ -42,12 +41,10 @@ backcall==0.2.0
     # via ipython
 beautifulsoup4==4.12.3
     # via nbconvert
-bleach[css]==6.1.0
-    # via
-    #   bleach
-    #   nbconvert
+bleach[css]==6.2.0
+    # via nbconvert
 bokeh==1.2.0
-    # via -r -
+    # via -r ./backend/src/apiserver/visualization/requirements.in
```

Comment:
We can't leave the full path because developers will run `pip-compile` from the directory where the `requirements.in` file is.
I think we can achieve the same behaviour in the CI by using [`working-directory`](https://github.com/kubeflow/pipelines/blob/cb07619ec6af0bfab980d4bcf172ff7ed39f2cad/.github/workflows/sdk-execution.yml#L56).

Reply:
ok, i'll fix it.

Reply:
done